vizro-core/src/vizro/_vizro.py

```diff
@@ -46,41 +59,18 @@ def __init__(self, **kwargs):
             use_pages=True,
         )
 
-        # Include Vizro assets (in the static folder) as external scripts and stylesheets. We extend self.dash.config
-        # objects so the user can specify additional external_scripts and external_stylesheets via kwargs.
-        vizro_assets_folder = Path(__file__).with_name("static")
-        requests_pathname_prefix = self.dash.config.requests_pathname_prefix
-        # Exclude vizro/css/figures.css since these are distributed through the vizro._css_dist mechanism.
-        # In future we will probably handle all assets this way and none of this code will be required.
-        vizro_css = [
-            requests_pathname_prefix + path
-            for path in self._get_external_assets(vizro_assets_folder, "css")
-            if path != "vizro/css/figures.css"
-        ]
-
-        # Ensure vizro-bootstrap.min.css is loaded in first to allow overwrites
-        vizro_css.sort(key=lambda x: not x.endswith("vizro-bootstrap.min.css"))
```

Comment:
This was mainly to ensure that if a user wants to overwrite any bootstrap class names, this still works 👍 I'll test it out manually later, but it's better to have this loaded in first.

Reply:
Got it, thank you! Made it so that `vizro-boostrap.min.css` is first in [ad943ab](https://github.com/mckinsey/vizro/pull/775/commits/ad943ab6807779652ddc367bdde7f47a49295046) and screenshots are now passing.

The full order is now:
1. vizro library assets in arbitrary order
2. vizro framework assets with `vizro-boostrap.min.css` first and rest in arbitrary order
3. user assets - not sure what order

Note that your example about `figures.css` won't actually work since as the moment library assets are always before the framework ones. We could probably change this if we needed to but since it doesn't cause any issues and might never do I'll leave it.

![image](https://github.com/user-attachments/assets/2c24f949-fe39-4640-8324-037e26bff3be)

Reply:
Alright, shall we read in things in alphabetical order (except for the bootstrap stylesheet)? This is what Dash does under the hood when reading the user assets folder

Reply:
Yes, good idea! Done in [efef57e](https://github.com/mckinsey/vizro/pull/775/commits/efef57e0fb29c26393041891cc94dab4987260d8).

Now the order is:

1. vizro library assets in alphabetical order
2. vizro framework assets with vizro-boostrap.min.css first and rest in alphabetical order
3. user assets in alphabetical order